ocw/lib/cleanup.py

```diff
@@ -25,6 +26,9 @@ def cleanup_run():
             if ProviderChoice.GCE in providers:
                 GCE(namespace).cleanup_all()
 
+            if ProviderChoice.Openstack in providers:
```

Comment:
looks like you forgot to update enum with new entry ( at least I fail to find this in PR )